package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "solution-json",
+  "version": "1.0.0",
+  "description": "Has everything neededto create gifs",
+  "main": "testGIF.js",
+  "dependencies": {
+    "canvas": "^1.3.16",
+    "minimist": "^1.2.0"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "node main.js"
```

Comment:
look up node package.json and how you can add scripts here. Generally all programs are run by executing npm start in which in this case would execute node testGif.js